npc/re/quests/quests_dungeons_200.txt

```diff
@@ -0,0 +1,489 @@
+	case 1:
+		emotion ET_SLEEPY;
+		mes "[Pion]";
+		mes "Zzz... Whoa!";
+		mes "Excuse me! No more work!";
+		next;
+		mes "[Pion]";
+		mes "Oops... ah, this is definitely not saliva";
+		mes "It is a drop of sweat that shines with passion and effort.";
+		next;
+		mes "[Pion]";
+		mes "Hmm, hmmm. The entrance down this slope is now restricted...";
+		mes "What's going on?";
+		next;
+		if (select( "I want to go to the 4th floor of Rudus.", "Point out the working attitude." ) == 2) {
+			mes "[Pion]";
+			mes "Oh, there's a problem with your working attitude?";
+			mes "Sa, some people might sleep!!";
+			next;
+			mes "[Pion]";
+			mes "I only slept a little!";
+			mes "Really!";
+			close;
+		}
+		mes "[Pion]";
+		mes "Fourth floor? I'm in control right there...";
+		mes "Ah, did you come to see Yulma from Bullet Meow?";
+		next;
+		mes "[Pion]";
+		mes "Then you should pass it.";
+		mes "If you want to look at the specimens on the 4th floor and help with a research request, go back to <NAVI>[Yulma]<INFO>pub_cat,94,43,0,101,0</INFO></NAVI> and ask.";
+		next;
+		mes "[Pion]";
+		mes "I'll move you to the 4th floor.";
+		close2;
+		completequest 16521;
+		setquest 16522;
+		warp "sp_rudus4",370,156;
+		end;
+	case 2:
+		break;
+	}
+	emotion ET_SLEEPY;
+	mes "[Pion]";
+	mes "Zzz... Huh! Shi, excuse me!";
+	mes "Are you sure you want to go down to the 4th floor of Rudus?";
+	next;
+	if (select( "Move", "Do not move" ) == 2) {
+		mes "[Pion]";
+		mes "I see. Ha-am....";
+		close;
+	}
+	mes "[Pion]";
+	mes "I'll move you. Take care!";
+	close2;
+	warp "sp_rudus4",370,156;
+	end;
+
+OnInit:
+	questinfo( QTYPE_DAILYQUEST, QMARK_YELLOW, "isbegin_quest(16521) == 1" );
+	end;
+}
+
+sp_rudus4,192,204,3	script	Secret Wing Researcher#rudu	4_SOIL,{
+	if (isbegin_quest(16528) == 1) {
+		mes "[Secret Wings Researcher]";
+		mes "Good. Field work.";
+		mes "You need to observe a lot of monsters.";
+		next;
+		emotion ET_HUK;
+		mes "[Secret Wings Researcher]";
+		mes "...huh!";
+		mes "How did you recognize me?";
+		next;
+		mes "[Secret Wings Researcher]";
+		mes "Oh, look again, " + strcharinfo(0) + ".";
+		mes "I got a call. Are you here to renew the chip?";
+		next;
+		mes "[Secret Wing Researcher]";
+		mes "Let's see... Let's get some chips.";
+		mes "Hmmmm... Connect with my machine... Empty data...";
+		next;
+		emotion ET_BEST;
+		mes "[Secret Wing Researcher]";
+		mes "Okay, that's it! The data is clean and very nice. I'll paste it again.";
+		next;
+		mes "[Secret Wing Researcher]";
+		mes "Because this is also a job, <NAVI>[Yulma]<INFO>pub_cat,94,43,0,101,0</INFO></NAVI> will give you a small allowance when you return.";
+		mes "I am always indebted to you. Good job!";
+		completequest 16528;
+		setquest 16540;
+		close;
+	}
+	mes "[Secret Wing Researcher]";
+	mes "Good. Field work.";
+	mes "You have to watch a lot of monsters.";
+	next;
+	emotion ET_HUK;
+	mes "[Secret Wing Researcher]";
+	mes "...huh!";
+	mes "How did you recognize me?";
+	next;
```

Comment:
looks like this part should be above the "if", as it is the same as inside the if